### ISSUE TITLE
Study builder > Email template issues 

### ISSUE THREAD

**Issue by Lathan-btc** [GoogleCloudPlatform/fda-mystudies#2493]
Study builder > Email template issues 
1. Access link should be present next to the 'Link' title
2. Remove the prior at end of the subject 
3. Make Org Name to be consistent for all the mail templates ( For some it is displaying 'Org Name' and for some mails, it is displaying 'Organization' )

![template1](https://user-images.githubusercontent.com/71445210/102585932-3d247580-412f-11eb-87c6-220e3ac8e424.png)

![template2](https://user-images.githubusercontent.com/71445210/102585971-53323600-412f-11eb-83d4-59f7245b8bca.png)

**Comment by ESWAR1994** [comment]
I pushed my changes to QA branch ... need to test in QA instance

**Comment by ESWAR1994** [comment]
my changes are not reflecting and i will let you know once it is done

**Comment by aswinijena100** [comment]
@Lathan-btc Please test in QA

**Comment by ESWAR1994** [comment]
merged into develop branch need to test in dev

**Comment by aswinijena100** [comment]
@Lathan-btc please test in dev and close the issue if fixed

**Comment by Lathan-btc** [comment]
Issue is still reproducible in the latest Dev instance.


![templates1](https://user-images.githubusercontent.com/71445210/106263384-4d5d6380-624a-11eb-953e-2a89d47ef69f.png)

**Comment by harisboston** [comment]
@Lathan-btc `Organization's` is set in `orgName` placeholder.
Kindly let us know if its not consistent for all mails used in study builder or if this need to be changed to `org Name`.

**Comment by Lathan-btc** [comment]
Issue is fixed in the latest Dev instance.

**Comment by Lathan-btc** [comment]
@ESWAR1994 Issue is still reproducible in the latest automated deployment QA branch

 'MyStudies_Org_Name' should be changed to 'Organization'

![email template](https://user-images.githubusercontent.com/71445210/107001370-84ce9180-67af-11eb-8036-9fe4d1184c94.png)

**Comment by harisboston** [comment]
@Lathan-btc Fix is deployed in latest automated deployment QA branch, kindly test and update this ticket.

**Comment by Lathan-btc** [comment]
Issue is fixed in the latest QA instance.
